pkg/config/config.go

```
@@ -134,13 +132,27 @@ func (cfg *Config) SetupLogger() {
 	ctrlrt.SetLogger(zap.New(zap.UseFlagOptions(&zapOptions)))
 }
 
+// SetAWSAccountID uses sts GetCallerIdentity API to find AWS AccountId and set
+// in Config
+func (cfg *Config) SetAWSAccountID() error {
+	// use sts to find AWS AccountId
+	session := session.Must(session.NewSession())
```

`session.Must` will panic if `session.NewSession` returns an error. Maybe just return a wrapped error like: `fmt.Errorf("cannot create session: %v", err)`